R/ds.studentTestDP.R

```diff
@@ -18,6 +18,8 @@ source("R/utils.R")
 #' @return \code{ds.studentTestDP} returns a differentially private student-test
 #' @export
 
+delta = 0.01
```

Comment:
Maybe set that to a smaller value because I think that the statistical significance threshold for student tests is 0.01-0.1

Reply:
10e-4 should be decent enough!